app/Models/DailyChallengeUserStats.php

```diff
@@ -95,6 +95,30 @@ public function user(): BelongsTo
         return $this->belongsTo(User::class, 'user_id');
     }
 
+    public function fix(): void
+    {
+        $highScores = PlaylistItemUserHighScore
+            ::where('user_id', $this->user_id)
+            ->whereRelation('playlistItem.room', 'category', 'daily_challenge')
+            ->with('playlistItem.room')
+            ->orderBy('created_at')
+            ->get();
```

Comment:
This seems to imply only only days with plays are considered, so it's possible for the last play to be several days previously and it'll still have a current streak when it shouldn't?

Reply:
it'll eventually be fixed by the daily job